docs/operating/manuals/collaset.md

```diff
@@ -436,4 +434,125 @@ NAME                     READY   STATUS    RESTARTS     AGE
+
+# Once the created Pod is available for service, the old Pod will be deleted
+$ kubectl -n default get pod
+NAME                    READY   STATUS              RESTARTS   AGE
+collaset-sample-rcmbv   1/1     Running             0          1s
+collaset-sample-dwkls   1/1     Terminating         0          7m12s
+```
+
+The two Pods will have a pair of labels to identify their relationship. The new Pod will have the label `collaset.kusionstack.io/replace-pair-origin-name` to indicate the name of the old Pod, and the old Pod will have the label `collaset.kusionstack.io/replace-pair-new-id` to indicate the instance ID of the new Pod.
+
+Additionally, the new Pod and old Pod will each begin their own PodOpsLifecycles, which are independent of each other.
+
+### Recreate And Replace Specified Pod
+
+In practice, users often need to recreate or replace specified Pods under a CollaSet.
+
+To delete a Pod, users can simply call the Kubernetes API, like executing `kubectl delete pod <pod-name>`. 
+However, this will bypass the [PodOpsLifecycle](https://www.kusionstack.io/docs/operating/concepts/podopslifecycle) Mechanism. 
+We provide following two options:
+
+1. Enable the feature `GraceDeleteWebhook` so that it is possible to delete Pods through `PodOpsLifecycle`.
+```shell
+# Enable the GraceDeleteWebhook feature when starting the controller with this argument
+$ /manager --feature-gates=GraceDeleteWebhook=true
+```
+```shell
+$ kubectl -n default get pod
+NAME                    READY   STATUS        RESTARTS   AGE
+collaset-sample-vqccr   1/1     Running       0          21s
+
+# Delete the pod directly. A message will respond indicating that the Pod deletion is handled by PodOpsLifecycle
+kubectl -n default delete pod collaset-sample-vqccr
+Error from server (failed to validate GraceDeleteWebhook, pod deletion process is underway and being managed by PodOpsLifecycle): admission webhook "validating-pod.apps.kusionstack.io" denied the request: failed to validate GraceDeleteWebhook, pod deletion process is underway and being managed by PodOpsLifecycle
+
+# The old Pod is deleted, and a new Pod will be created 
+$ kubectl -n default get pod -w
+collaset-sample-vqccr   1/1     Terminating   0          71s
+collaset-sample-nbl6t   0/1     Pending       0          0s
+collaset-sample-nbl6t   0/1     Pending       0          0s
+collaset-sample-vqccr   1/1     Terminating   0          71s
+collaset-sample-nbl6t   0/1     ContainerCreating   0          0s
+collaset-sample-nbl6t   0/1     ContainerCreating   0          0s
+collaset-sample-nbl6t   1/1     Running             0          0s
+collaset-sample-nbl6t   1/1     Running             0          0s
+collaset-sample-vqccr   1/1     Terminating         0          72s
+collaset-sample-vqccr   0/1     Terminating         0          73s
+collaset-sample-vqccr   0/1     Terminating         0          73s
+collaset-sample-vqccr   0/1     Terminating         0          73s
+```
```

Comment:
I don't think using `-w` in the documentation is a good way to demonstrate.
This looks easier to understand: 
```
$ kubectl -n default get pod
collaset-sample-vqccr   1/1     Terminating   0          71s
collaset-sample-nbl6t   0/1     ContainerCreating   0          0s
```